ocaml/xapi/xapi_bond.ml

```diff
@@ -382,7 +382,9 @@ let create ~__context ~network ~members ~mAC ~mode ~properties =
 
       (* Set the PIF.bond_slave_of fields of the members.
          		 * The value of the Bond.slaves field is dynamically computed on request. *)
-      List.iter (fun slave -> Db.PIF.set_bond_slave_of ~__context ~self:slave ~value:bond) members;
+      (* Also set the per bridge IGMP Snooping status *)
+      List.iter (fun slave -> Db.PIF.set_bond_slave_of ~__context ~self:slave ~value:bond;
+                              Db.PIF.set_igmp_snooping_enabled ~__context ~self:slave ~value:`na) members;
```

Comment:
Let's not do this here, and make sure that the new field is only ever updated from one place: in nm.ml. The value of the field for bond slave does not matter very much, because the `currently_attached` field for those PIFs is always `false`.